test/src/examples_complete_test.go

```diff
@@ -46,17 +49,46 @@ func TestExamplesComplete(t *testing.T) {
   terraform.InitAndApply(t, terraformOptions)
 
   // Run `terraform output` to get the value of an output variable
-  // id := terraform.Output(t, terraformOptions, "id")
-  // arn := terraform.Output(t, terraformOptions, "arn")
-  // cluster_endpoint := terraform.Output(t, terraformOptions, "cluster_endpoint")
+  id := terraform.Output(t, terraformOptions, "id")
+  assert.Contains(t, id, "eg-ue2-test-memorydb-" + randID)
```

Comment:
Use `Equal` instead of `Contains` where appropriate.

```suggestion
  assert.Equal(t, "eg-ue2-test-memorydb-" + randID, id)
```